classes/Validate.php

```diff
@@ -615,7 +615,7 @@ public static function isEan13($ean13)
      */
     public static function isIsbn($isbn)
     {
-        return preg_match(Tools::cleanNonUnicodeSupport('/^[^<>;={}]*$/u'), $isbn);
+        return $isbn || preg_match('/^[0-9-]{0,32}$/', $isbn);
```

Comment:
oh I see what you mean, there's a typo here, I forgot the "!".
I'll fix it thanks !